### PR TITLE
add batch to orchestrate runner

### LINKED ISSUE #12556: [Feature Request] Batch mode in orchestrate runner
It would be good to have the batch mode in orchestrate runner's "state specifications".

For example - the following would run the `update_nginx_config` on only 2 minions matching the "web*" glob.

```
webserver_update:
  salt.state:
    - tgt: 'web*'
    - batch: 2
    - sls:
      - update_nginx_config
```

This would make upgrading code a very simple process through salt.

Is this perhaps already possible? I could not find it anywhere in the documentation.

### PR DESCRIPTION
**Pull request by gtmanfred**
closes #12556

### CODE REVIEW

**Comment by thedrow** [comment]
Can you add a test that verifies that this works as expected?

**Comment by gtmanfred** [comment]
I have tested it manually, I have no clue where to put it inside of the saltstack tests, so I would need someone from the QE team to give me an idea for that, or if they wanted to do it <3

**Comment by cachedout** [comment]
Since this has been tagged as needing tests, QA will put it on their list of tests to write. I'm fine with merging this as it sits, since it will be going into the develop branch.

File: salt/states/saltmod.py [hunk 1]
```diff
@@ -371,6 +379,7 @@ def function(
                 ).format(fun, tgt, str(False))
         ret['result'] = None
         return ret
+    cmd_ret = __salt__['saltutil.cmd'](tgt, fun, **cmd_kw)
```
Comment (khrisrichardson) [inline salt/states/saltmod.py]: @gtmanfred @cachedout this looks redundant in light of the try block below

Comment (gtmanfred) [inline salt/states/saltmod.py]: That's my fault, I added the extra one outside of the try block to see a stack trace of an exception, that extra one can be removed.

**Comment by nleib** [comment]
what version  should this feature be out?

**Comment by jfindlay** [comment]
@nleib, this will be coming out in Salt Boron.

**Pull request merged**